src/Componet/Header.jsx

```diff
@@ -1,24 +1,36 @@
 import React, { useState } from 'react'
-import { Link} from "react-router-dom";
-import {FaInstagram, FaLinkedin} from "react-icons/fa"
+import { Link } from "react-router-dom";
+import { FaInstagram, FaLinkedin } from "react-icons/fa"
 import "./header.css"
 import Register from '../Register';
 
 export default function Header() {
-    const[Open,setOpen]= useState(false);
+  const [Open, setOpen] = useState(false);
```

Comment:
pra seguir o padrão do mercado, nomes de variáveis devem começar com letra minúscula

```suggestion
  const [open, setOpen] = useState(false);
```